qatrfm/environment.py

```diff
@@ -27,15 +27,16 @@
 from qatrfm.domain import Domain
 from qatrfm.utils.logger import QaTrfmLogger
 from qatrfm.utils import libutils
+from pathlib import Path
```

Comment:
please move this import line to the first block, as it belongs to the system default packages